README.md

```diff
@@ -1,56 +1,46 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Kanban board: Car collector
 
+> Customers who wish to rent a car can use CarCollectors' website. Using the Vehicles API, the visitor may explore a vast selection of stunning cars. It also allows users to like, comment on, and reserve a certain automobile. The Involvement API is used to save all data.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- html
+- css
+- javascript
+- jest
+- webpack
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://rizwan-learn.github.io/mod2-Kanban/) <br>
+[Live Demonstration Link](https://www.youtube.com/watch?v=mR17kJi45D0)
 
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
+- Git clone
+- npm install
 
 ### Run tests
+npm test
 
 ### Deployment
-
-
+Gh-pages
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@regiss05](https://github.com/regiss05)
+- Twitter: [@regiss05](https://twitter.com/regissmukubiza)
+- LinkedIn: [regiss05](https://www.linkedin.com/in/regiss-mukubiza-1bab841b3/)
 
 👤 **Author2**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@rizwanhasan](https://github.com/Rizwan-learn/)
+- Twitter: [@rizwanhasan](https://twitter.com/twitterhandle)
+- LinkedIn: [rizwanhasan](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] [OPTIONAL] I think it would be better if you could personalize the MIT license on your file. I will leave the places that need to be changed. 

<img width="1048" alt="Screen Shot 2022-04-16 at 5 23 52 PM" src="https://user-images.githubusercontent.com/69011963/163675002-27479dfa-b385-450c-946f-1d593cf9d1be.png">